.eslintrc

```diff
@@ -211,60 +212,55 @@
       ["error", "never"],
     "array-element-newline": 0,            // enforce line breaks after each array element
-    "block-spacing": 0,                    // enforce consistent spacing inside single-line blocks
+    "block-spacing": "error",                    // enforce consistent spacing inside single-line blocks
     "brace-style":                         // enforce consistent brace style for blocks
       ["error", "stroustrup"],
     "camelcase": 0,                        // enforce camelcase naming convention
     "capitalized-comments": 0,             // enforce or disallow capitalization of the first letter of a comment
-    "comma-dangle": ["error", "never"],    // require or disallow trailing commas
+    "comma-dangle": "error",               // require or disallow trailing commas
     "comma-spacing": 0,                    // enforce consistent spacing before and after commas
-    "comma-style": ["error", "last"],      // enforce consistent comma style
+    "comma-style": "error",                // enforce consistent comma style
     "computed-property-spacing": 0,        // enforce consistent spacing inside computed property brackets
     "consistent-this": 0,                  // enforce consistent naming when capturing the current execution context
-    "eol-last": 0,                         // require or disallow newline at the end of files
-    "func-call-spacing": 0,                // require or disallow spacing between function identifiers and their invocations
-    "func-name-matching": 0,               // require function names to match the name of the variable or property to which they are assigned
+    "eol-last": "error",                   // require or disallow newline at the end of files
+    "func-call-spacing": "error",          // require or disallow spacing between function identifiers and their invocations
+    "func-name-matching": "error",         // require function names to match the name of the variable or property to which they are assigned
     "func-names": 0,                       // require or disallow named function expressions
-    "func-style": ["error", "expression"], // enforce the consistent use of either function declarations or expressions
+    "func-style": "error",                 // enforce the consistent use of either function declarations or expressions
     "function-paren-newline": 0,           // enforce consistent line breaks inside function parentheses
     "id-blacklist": 0,                     // disallow specified identifiers
     "id-length": 0,                        // enforce minimum and maximum identifier lengths
     "id-match": 0,                         // require identifiers to match a specified regular expression
     "indent": ["error", 4, {               // enforce consistent indentation
       "ignoredNodes": [
-        ":nth-child(1)"
+        ":nth-child(1)",
+        ":nth-child(2)"
```

Comment:
this allows us to deal with the the requirejs/angular boilerplate in a more palatable fashion